src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.ts

```diff
@@ -18,7 +21,13 @@ export class UbsBaseSidebarComponent implements AfterViewInit {
   @ViewChild('drawer') drawer: MatDrawer;
   @ViewChild('sidebarContainer') sidebarContainer: ElementRef;
 
-  constructor(public breakpointObserver: BreakpointObserver) {}
+  constructor(public serviceUserMessages: UserMessagesService, public breakpointObserver: BreakpointObserver) {}
+
+  ngOnInit(): void {
+    this.serviceUserMessages
+      .getCountUnreadNotification()
+      .subscribe((response) => (this.serviceUserMessages.countOfNoReadeMessages = response));
```

Comment:
we have to add pipe and use takeUntil(this.destroy) or take(1) for unsubscribe
https://blog.bitsrc.io/6-ways-to-unsubscribe-from-observables-in-angular-ab912819a78f
or check our code

Reply:
Done